packages/core/saved-objects/core-saved-objects-utils-server/src/index.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export { mergeSavedObjectMigrationMaps } from './merge_migration_maps';
+export { SavedObjectsErrorHelpers, type DecoratedError } from './saved_objects_error_helpers';
+export {
+  SavedObjectsUtils,
+  ALL_NAMESPACES_STRING,
+  DEFAULT_NAMESPACE_STRING,
+  FIND_DEFAULT_PAGE,
+  FIND_DEFAULT_PER_PAGE,
```

Comment:
These constants are in practice only used internally by Core, **but** they are also used by the `SavedObjectsUtils` implementation that is publicly exposed. To avoid duplicating them, I exposed them from the public util package, as it didn't felt like it would be causing any harm for them to be publicly accessible.